framework/generated/generated_vulkan_state_table.h

```diff
@@ -313,6 +313,375 @@ class VulkanStateTable : VulkanStateTableBase
     std::map<format::HandleId, VideoSessionParametersKHRWrapper*> videoSessionParametersKHR_map_;
```

Comment:
This is a very superficial comment and I don't understand the architecture here, but are we maintaining these original maps and the unordered ones below for every handle live in the system at any one time?

E.g. every image created will have its handle tracked in `std::map<format::HandleId, ImageWrapper*> image_map_` and also in `std::unordered_map<VkImage, ImageWrapper*> image_map_;` below at the same time for the lifetime of the image?